src/__Private/codegen/CodegenEditableTokenFromData.php

```diff
@@ -23,6 +26,29 @@ public function generate(): void {
 
     $tokens = $this->getSchemaTokens();
 
+    $class_map = dict[];
+    $class_map_with_text = dict[];
+
+    foreach ($tokens['noText'] as $token) {
+      $kind = self::underscored($token['token_kind']);
+      $class_map[$kind] = sprintf('HHAST\\%sToken::class', $token['token_kind']);
+    }
+
+    foreach ($tokens['fixedText'] as $token) {
+      $text = TypeAssert\not_null($token['token_text']);
+      $kind = $token['token_kind'];
+      if (Str\lowercase($text) === Str\uppercase($text)) {
+        $class_map[$text] = sprintf('HHAST\\%sToken::class', $kind);
+      } else {
+        $class_map_with_text[$text] = sprintf('HHAST\\%sToken::class', $kind);
+      }
+    }
+
+    foreach ($tokens['variableText'] as $token) {
+      $kind = self::underscored($token['token_kind']);
```

Comment:
whoops, I should move this to _Private\StrP like upper_camel

Reply:
I can include the move on the next PR update!